library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/MaskAllWireframeMapperTest.kt

```diff
@@ -180,4 +183,32 @@ internal class MaskAllWireframeMapperTest {
     fun `M return the ImageMapper W getImageMapper`() {
         assertThat(testedMaskAllWireframeMapper.imageMapper).isEqualTo(mockImageWireframeMapper)
     }
+
+    @Test
+    fun `M delegate to DecorViewWireframeMapper W map { view with no parent }`() {
+        // Given
+        val mockView: View = mockViewWithNoViewTypeParent()
```

Comment:
probably it should be the opposite: `mockViewWithEmptyParent` here and `mockViewWithNoViewTypeParent` in the test below

Reply:
yes...good spot